tests/failpoints/cases/test_pd_client.rs

```diff
@@ -142,3 +142,31 @@ fn test_slow_periodical_update() {
     fail::remove(pd_client_reconnect_fp);
     handle.join().unwrap();
 }
+
+// Reconnection will be speed limited.
+#[test]
+fn test_reconnect_limit() {
+    let pd_client_reconnect_fp = "pd_client_reconnect";
+    let server = MockServer::new(1);
+    let eps = server.bind_addrs();
+
+    let mut cfg = new_config(eps);
+    let env = Arc::new(EnvBuilder::new().cq_count(1).build());
+    let mgr = Arc::new(SecurityManager::new(&SecurityConfig::default()).unwrap());
+    cfg.update_interval = ReadableDuration(Duration::from_secs(100));
+    let client = RpcClient::new(&cfg, Some(env), mgr).unwrap();
+
+    // The REQUEST_RECONNECT_INTERVAL is 1s so sleeps 2s here.
+    thread::sleep(Duration::from_secs(2));
```

Comment:
Why sleep?

Reply:
The initial value of `last_try_connect` is the time when the client is created. Reconnection within 100ms will be canceled and an error will be returned.